policies/rubykaigi2017/accessibility-policy-en.md

```diff
@@ -0,0 +1,13 @@
+# Accessibility
+
+At RubyKaigi, we want you to be happy.  We value your attendance.
+
+As such, organizers are dedicated to providing conference experiences and environments that are equally welcoming, inviting, and accessible for all attendees. Some steps to ensure this are built into the program and physical design of the conference:
+
+  - We chose a conference venue that is barrier-free, proven to be accessible by wheelchairs.
+  - We provide lactation room, as well as an onsite daycare center engaging national-licensed childcare professionals.
+  - We offer significantly discounted tickers as well as admission free day for students.
+  - We provide invitation letters to whom need to obtain a visa in order to visit Japan.
+  - For those who cannot fly to the venue, all sessions will be recorded and made available on the website after the event.
+
+However, the organizers also recognize that accessibility issues are diverse. If you have any other accessibility requirements, or if you have any other feedback or questions, then please don’t hesitate to [get in touch with us](mailto:info@rubykaigi.org). While we can't promise the ability to accommodate each individual situation, we absolutely commit to listening to every concern and request with an open ear.
```

Comment:
This is not grammatically wrong, but I would reword the last sentence a bit:

"While we can't promise the ability to accommodate each individual situation, we absolutely commit to listening to..." ->
"Whiel we cannot promise to accommodate each individual situation, we are committed to listening to..."